Updated to the new UI (#1529)
keycloak/keycloak-documentation: authorization_services/topics/resource-server-enable-authorization.adoc

[[_resource_server_enable_authorization]]
= Enabling authorization services

ifeval::[{project_community}==true]

> To turn your OIDC Client Application into a resource server and enable fine-grained authorization, click the *Authorization Enabled* switch to *ON* then click *Save*.

Reviewer: ```suggestion
You can turn your OIDC client into a resource server and enable fine-grained authorization.

 .Procedure
  . Toggle *Authorization Enabled* to *ON*.
  . Click *Save*.
```

endif::[]
ifeval::[{project_product}==true]

> To turn your OIDC Client Application into a resource server and enable fine-grained authorization, select *Access type* *confidential* and click the *Authorization Enabled* switch to *ON* then click *Save*.

Reviewer: ```suggestion
You can now turn your OIDC client into a resource server and enable fine-grained authorization.
 .Procedure
  . Set *Access type* to *confidential*.
  . Toggle *Authorization Enabled* to *ON*.
  . Click *Save*.
```

endif::[]

> 

Reviewer: ```suggestion
+
```

.Enabling authorization services

> image:{project_images}/resource-server/client-enable-authz.png[alt="Enabling authorization services"]

Reviewer: ```suggestion
image:{project_images}/resource-server/client-enable-authz.png[Enabling authorization services]
```
